setup.py

```diff
@@ -6,16 +6,16 @@
 
 
 #: The name of the package on PyPi
-PYPI_PACKAGE_NAME = 'pynput'
+PYPI_PACKAGE_NAME = 'oa_pynput'
 
 #: The name of the main Python package
-MAIN_PACKAGE_NAME = 'pynput'
+MAIN_PACKAGE_NAME = 'oa_pynput'
 
 #: The package URL
-PACKAGE_URL = 'https://github.com/moses-palmer/pynput'
+PACKAGE_URL = 'https://github.com/OpenAdaptAI/pynput'
 
 #: The author email
-AUTHOR_EMAIL = 'moses.palmer@gmail.com'
+AUTHOR_EMAIL = 'richard.abrich@mldsai.com'
```

Comment:
Let's leave this as it was

Reply:
addressed in https://github.com/OpenAdaptAI/pynput/pull/3/commits/7e36c55e2ea0ccc47d0a8ca37487f002edd0027f